Tighten up terminology definitions. (#141)
ietf-rats-wg/architecture: draft-ietf-rats-architecture.md

Appraisal Policy for Evidence:

> : A set of rules, reference values and such that is used by a Verifier to evaluate the Evidence from an Attester. Compare /security policy/ in {{RFC4949}}

Reviewer: I liked the previous phrasing better. E.g. +1 "informs", -1 "and such"

Reviewer 2: "A set of rules that informs a Verifier regarding the evaluation of the Evidence." or "A set of rules that informs a Verifier regarding the appraisal of the Evidence."

Reviewer 2: Note use of 'appraisal' in section below on Attestation. Should use common wording if we mean similar things.

Author: Ned's wording plus mention of reference values: "A set of rules and optional reference values that informs a Verifier regarding the appraisal of the Evidence." Mention of reference values will greatly help the reader make sense quicker IMO.

Reviewer: ```suggestion
: A set of rules that informs a Verifier regarding the appraisal of the Evidence, e.g., "byte-compare in little-endian Claims with reference values in big-endian". Compare /security policy/ in {{RFC4949}}.
```

Appraisal Policy for Attestation Results:

: A set of rules that direct how a Relying Party uses the Attestation Results regarding an Attester generated by the Verifiers. Compare /security policy/ in {{RFC4949}}

Attestation Result:

> : The output generated by a Verifier, typically including information about an Attester, where the Verifier vouches for the validity of the results.

Reviewer: ```suggestion
: The output generated by a Verifier, typically including information about an Attester, where the Verifier vouches for the validity of the results
```

Reviewer: not a sentence (applies multiple times)

Reviewer 2: "The output generated by a Verifier, that typically includes information about the Attester, that is meaningful to a Relying Party and where the Verifier vouches for the validity of the Attestation Results."

Author: Tightening up Ned's text further: "The output generated by a Verifier, that is information about the Attester, that is used by a Relying Party and where the Verifier vouches for the Attestation Results."

Reviewer: ```suggestion
: The output generated by a Verifier. Attestation Results are information about an Attester that is used by a Relying Party and where the Verifier vouches for the Attestation Results.
: The output generated by a Verifier, typically including information about an Attester, where the Verifier vouches for the validity of the results.
```

Attester:

> : A role performed by an entity (typically a device) whose Evidence must be appraised in order to infer the extent to which the Attester is considered trustworthy, such as when deciding whether it is authorized to perform some operation.

Reviewer: ```suggestion
: A role performed by an entity (typically a device) whose Evidence must be appraised in order to infer the extent to which the Attester is considered trustworthy, such as when deciding whether it is authorized to perform some operation
```

Reviewer 2: +1

Author: +1

> Claim:
> : A piece of asserted information, often in the form of a name/value pair.
> (Compare /claim/ in {{RFC7519}})

Reviewer: ```suggestion
Claim:
: A piece of asserted information, often in the form of a name/value pair
(Compare /claim/ in {{RFC7519}})
```

Reviewer 2: Should have a period after {{RFC7519}}) since it was removed from 'pair'

Reviewer: Aligned with Appraisal Policy and adapted use of period again

Endorsement:

> : A secure statement that an Endorser (typically a manufacturer) vouches for the integrity of an Attester. It may include implied Claims.

Reviewer: ```suggestion
: A secure statement that an Endorser (typically a manufacturer) vouches for the integrity of an Attester. It may include implied Claims
```

> : A secure statement that an Endorser (typically a manufacturer) vouches for the integrity of an Attester. It may include implied Claims.

Reviewer 3: The term "implied" Claims is not defined and is confusing in my opinion.

Endorser:

> : An entity (typically a manufacturer) whose Endorsements are required for Verifiers to appraise the authenticity of Evidence.

Reviewer: ```suggestion
: An entity (typically a manufacturer) whose Endorsements are required for Verifiers to appraise the authenticity of Evidence
```

Relying Party Owner:

> : An entity (typically an administrator), that is authorized to configure Appraisal Policy for Attestation Results in a Relying Party.

Reviewer: ```suggestion
: An entity (typically an administrator), that is authorized to configure Appraisal Policy for Attestation Results in a Relying Party
```

Reviewer 2: Maybe there is a convention for not including period at the end of a sentence?

Reviewer 3: RFC 4949 which we reference, uses periods.

Verifier:

: A role performed by an entity that appraises the validity of Evidence about an Attester

>  and produces Attestation Results to be used by a Relying Party.

Reviewer: ```suggestion
 and produces Attestation Results to be used by a Relying Party
```

Reviewer 2: 'that' instead of 'and' produces

Verifier Owner:

> : An entity (typically an administrator), that is authorized to configure Appraisal Policy for Evidence in a Verifier.

Reviewer: ```suggestion
: An entity (typically an administrator), that is authorized to configure Appraisal Policy for Evidence in a Verifier
```
